Prog/LgUnit - Ipums Table 2B.sas

```diff
@@ -147,6 +147,16 @@ data vacdata;
 			else if rent > 0 then vac = 1;
 		if vac = 1;
 	%end;
+
+	%else %if &ten. = ALL %then %do;
```

Comment:
@rpitingolo - i'm not sure i'm following this code here. why if there is a value greater than 0 or rent greater than 0 would it also be vac=1? (also the rent line repeats rent? is that just a flag to include the observation in the dataset (vs assigning a value to a variable)?

Reply:
@lhendey I'll take a look. I think I added this last Friday when trying to get the vacancy rate stuff working again.